### PR TITLE
Override focus outline

### PR DESCRIPTION
**Pull request by marklchaves**
Override default browser focus outline on .pum-content to none.

## Description

`
.pum-content:focus {
	outline: none !important;
}
`

Related Issue:

https://github.com/PopupMaker/Popup-Maker/issues/970

## Types of changes

CSS

## Screenshots

popup-maker-fix-for-970-unwanted-outline

## This has been tested in the following browsers

- [x] Chrome
- [x] Firefox
- [x] Edge
- [x] Safari
- [x] Opera

## Merge Checklist

- [ ] This PR passes all automated checks (will appear once pull request is submitted)
- [ ] My code has been tested in the latest version of WordPress.
- [ ] My code does not have any warnings from ESLint.
- [ ] My code does not have any warnings from StyleLint.
- [ ] My code does not have any warnings from PHPCS.
- [ ] My code follows [the WordPress coding standards](https://developer.wordpress.org/coding-standards/).
- [ ] My code follows [the accessibility standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/accessibility/).
- [ ] All new functions and classes have code documentation.

### CODE REVIEW

**Comment by danieliser** [comment]
@marklchaves You'll need to update it via the SCSS files, the CSS is all generated at build time.

https://github.com/PopupMaker/Popup-Maker/blob/master/assets/sass/partials/_pum_styles.scss#L166

**Comment by marklchaves** [comment]
Tested on WordPress 5.8.

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,5 @@ contributors.md
 !/packages/README.md
 
 /yarn-error.log
+assets/.DS_Store
```
Comment (danieliser) [inline .gitignore:34]: You might be able to a global .DS_Store ignore rule

**Review by danieliser — APPROVED** [review]
Not a big deal, but you don't have to commit the built css with your PR's going forward. We will do all of that officially when we create a release.

**Comment by marklchaves** [comment]
Hey @danieliser, let me know if I need to do anything else to get this CSS change into production. Thanks!

**Comment by danieliser** [comment]
@marklchaves Curious, is the !important necessary, or could it be dropped? Typically not a great idea to use it in a plugin as it leaves very little room to overload it in the users custom CSS for example.

But in this case I don't know that we would ever really want the content area to highlight, so I'm torn.

Either way I'll get it merged tonight.

**Comment by marklchaves** [comment]
Hey @danieliser,

I reran my baseline tests without `!important` and everything looked fine. I might have had some edge cases where I needed that before. I agree that it's misused a lot and I don't want to fall into that boat too.

Let's go without the `!important` and revisit/investigate if we need to later.

Thanks for checking!

TL;DR: remove `!important`.
